server/src/routes/handover.route.ts

```diff
@@ -0,0 +1,36 @@
+import { NextFunction, Request, Response } from "express";
+import Auth from '../middlewares/auth';
+import { HandoverRepository } from '../repository/handover.repository';
+import { HandoverService } from '../services/handover.service';
+
+import BasicRoute from "../bases/route.abstract";
+import HandoverController from "../controller/handover.controller";
+import CacheService from '../services/cache.service';
+
+export default class HandoverRoute extends BasicRoute {
+  constructor() {
+    super();
+    this.setPrefix("handover");
+    this.setRoutes();
+  }
+
+  protected setRoutes() {
+
+    const auth = new Auth();
+    this.router.get("/",  auth.authAdmin.bind(auth),(req, res, next) =>
+    HandoverController.getAll(req, res, next)
+    );
+    this.router.get("/:id", auth.authAdmin.bind(auth), (req, res, next) =>
+    HandoverController.getById(req, res, next)
+    );
+    this.router.post("/",  auth.authAdmin.bind(auth),(req, res, next) =>
```

Comment:
you can only handover using manager's account ?

Reply:
Anyone who is logged in can operate.

Reply:
Is it right to replace authAdmin with authUse

Reply:
yes, i think so
if you use authAdmin, means just admin can login in.